tgui/packages/tgui/interfaces/SlotMachine.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2020 @ZeWaka
+ * SPDX-License-Identifier: ISC
+ */
+
+import { useBackend } from '../backend';
+import { Box, Button, NoticeBox, Divider, BlockQuote, Icon } from '../components';
+import { Window } from '../layouts';
+
+export const SlotMachine = (props, context) => {
+  const { data } = useBackend(context);
+  const { scannedCard, busy } = data;
+  return (
+    <Window
+      title="Slot Machine"
+      width={375}
+      height={175}
+      resizable>
```

Comment:
Does it need to be resizable?

Reply:
I mean, why not?

Reply:
I don't think the content's dynamic in size (so the set width/height is probably always what you actually want), but that's fair I guess.